composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "php-pm/httpkernel-adapter",
     "license": "MIT",
     "require": {
-        "php-pm/php-pm": "^1.0",
+        "php-pm/php-pm": "^1.1",
```

Comment:
maybe `"php-pm/php-pm": "^1.1|dev-master",`  ?
<img width="1241" alt="screen shot 2018-03-22 at 2 24 59 pm" src="https://user-images.githubusercontent.com/2576509/37770359-d8473598-2ddc-11e8-8cdc-38c40ed451b0.png">

Reply:
Never, don't think so. Use branch aliases if you need dev-master.